packages/core/src/config.js

```diff
@@ -46,6 +46,20 @@ export const configSchema = {
           }]
         }
       },
+      disallowedHostnames: {
```

Comment:
You'll also need to add a reference around L140 in this same file if we want to support per-snapshot disallowed hostnames.